### DISCUSSION TITLE
How to install plugins

### DISCUSSION THREAD

**Discussion by SrinSS01** [NvChad/NvChad#1993] [General]
with [vim-plug](https://github.com/junegunn/vim-plug) we can just add the following line in `init.vim`
`Plug 'fatih/vim-go', { 'do': ':GoUpdateBinaries' }`
How to do the same in NVimChad?

**Comment by siduck** [comment]
@SrinSS01 we use lazy.nvim as package manager https://github.com/folke/lazy.nvim

https://nvchad.com/docs/config/plugins

**Comment by SrinSS01** [comment]
after installing the plugin when I run `:GoUpdateBinaries` as it was mentioned in `Plug 'fatih/vim-go', { 'do': ':GoUpdateBinaries' }`
![image](https://user-images.githubusercontent.com/60167747/236379986-f7f96d56-edcb-4fd3-b91a-b5c3f19afa3f.png)
This messages is displayed
![image](https://user-images.githubusercontent.com/60167747/236379938-497e958b-a0da-4fc2-8f94-7259b652c95f.png)

**Reply by siduck** [reply]
@SrinSS01  please read our doc binaries properly , 

Also its build: "GoUpdateBinaries" , in lazy.nvim , and add lazy=false
